infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -59,9 +60,19 @@ public class LogReplicationConfig {
             PROTOBUF_TABLE_ID
     ));
 
+    // LogReplicationConfigManager contains a suite of utility methods for updating LogReplicationConfig
+    private LogReplicationConfigManager configManager;
+
     // Unique identifiers for all streams to be replicated across sites
     private Set<String> streamsToReplicate;
 
+    // Mapping from stream ids to their fully qualified names.
+    private Map<UUID, String> streamsIdToNameMap;
+
+    // Only drop streams that have explicitly set is_federated flag to false
+    // If streams have not been opened (no records in registry table), they should still be applied
+    private Set<UUID> confirmedNoisyStreams;
```

Comment:
Do we still need the noisyStreams now? Can't we use the in-memory registry table?

Reply:
Removed noisy streams as per the cluster upgrade discussion.